CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
-## [*Ongoing*](https://github.com/pbrisbin/downgrade/compare/v6.3.0...master)
+## [*Ongoing](https://github.com/pbrisbin/downgrade/compare/v6.4.0...master)
```

Comment:
You've dropped the second `*` on this heading and the next, breaking the bolding.

Reply:
My bad. I will just remove the bolding since most of the older entries are not bold, and this is anyways a header.